packages/sanity/src/core/form/inputs/PortableText/PortableTextInput.tsx

```diff
@@ -342,15 +345,11 @@ export function PortableTextInput(props: PortableTextInputProps) {
           </PortableTextMemberItemsProvider>
         </PortableTextMarkersProvider>
       )}
+      <pre>{JSON.stringify(focusPath)}</pre>
```

Comment:
this should probably be removed before merge :)

Reply:
Hehe, yup I saw it and rebased it away right after I opened this PR up 😅